cclient/api/core/check_consistency.h

```diff
@@ -17,21 +26,15 @@ extern "C" {
 #endif
 
 /**
- * Checks if a transaction is _consistent_ or a set of transactions are
- * _co-consistent_.
- * As long as a transaction is consistent it might be accepted
- * by the network. In case transaction is inconsistent, it will not be accepted,
- * and a reattachment is required by calling iota_client_replay_bundle().
- *
- * https://iota.readme.io/reference#checkconsistency
+ * @brief Check the consistency of the subtangle formed by the provided tails.
  *
- * @param service IRI node end point.
- * @param req - Request containing tail transaction hash (hash of
- * transaction with `currentIndex=0`), or array of tail transaction hashes.
- * @param res - Response containing consistency state of given transaction
- * or co-consistency of given transactions.
+ * As long as a transaction is consistent it might be accepted by the network. In case a transaction is inconsistent, it
```

Comment:
Suggestion - 
An inconsistent transaction will not be accepted by the network,  and a reattachment is required by ....